db/schema.rb

```diff
@@ -415,6 +415,14 @@
     t.integer "ecstat4"
     t.integer "ecstat5"
     t.integer "ecstat6"
+    t.integer "pregyrha"
+    t.integer "pregla"
+    t.integer "pregghb"
+    t.integer "pregother"
+    t.integer "disabled"
+    t.integer "wheel_value_check"
+    t.integer "hhregres"
+    t.integer "hhregresstill"
```

Comment:
should these be included?

Reply:
I don't think so, try dropping and recreating the db, it should update it

Reply:
Cheers!